### ISSUE TITLE
Access violation reading location

### ISSUE THREAD

**Issue by russokiwi** [VirtualAlllocEx/Shellcode-Downloader-CreateThread-Execution#1]
When compiling with VS2019 getting a 

"Exception thrown at 0x00007FF91BA42717 (vcruntime140d.dll) in Shellcode-Download_CreateThread_Execution.exe: 0xC0000005: Access violation reading location 0x0000000000000000."

Could you shed some light ?

**Comment by VirtualAlllocEx** [comment]
Did you set in code generation, the **runtime library** to **Multi-threaded (/MT)**? 
![image](https://user-images.githubusercontent.com/50073731/163768884-ea67b3dc-932e-46e3-81d5-46679b1ffc82.png)

**Comment by russokiwi** [comment]
Yes, set to /MT - now getting a different error:  Exception thrown: read access violation.
**str1** was nullptr.

**Comment by VirtualAlllocEx** [comment]
I am sorry, I don't have an answer at the moment.

**Comment by Cyberdyne-Security** [comment]
_**I Have Done Some Tests and after setting it to /MT ,**_
Exception thrown at 0x0000025A694A1000 in Shellcode-Download_CreateThread_Execution.exe: 0xC0000005: Access violation executing location 0x0000025A694A1000.
